projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -349,6 +349,8 @@ private function build_task( $task ) {
 		$built_task['completed']    = $this->is_task_complete( $task );
 		$built_task['disabled']     = $this->is_task_disabled( $task );
 		$built_task['subtitle']     = $this->load_subtitle( $task );
+		$built_task['body']         = $this->load_value_from_callback( $task, 'body_callback' );
+		$built_task['body_context'] = isset( $task['body_context'] ) ? $task['body_context'] : array();
```

Comment:
What happens if we add a `body_callback` but no `body_context` -- does the callback get applied to all contexts by default? Do we need to do any kind of error checking around that?

Reply:
We use a filter on the front end to check if the current context is in the array, which means that it shouldn't show in any context if the `body_context` is an empty array.  I removed the definition in case the `body_context` is empty either way to prevent sending more data to the front end unnecessarily.